storage/innobase/include/ut0ut.h

```diff
@@ -88,15 +88,26 @@ struct ut_when_dtor {
    the YieldProcessor macro defined in WinNT.h. It is a CPU architecture-
    independent way by using YieldProcessor. */
 #  define UT_RELAX_CPU() YieldProcessor()
-# elif defined(HAVE_ATOMIC_BUILTINS)
+# elif defined(__powerpc__)
+#include <sys/platform/ppc.h>
 #  define UT_RELAX_CPU() do { \
-     volatile lint	volatile_var; \
-     os_compare_and_swap_lint(&volatile_var, 0, 1); \
```

Comment:
But when HAVE_ATOMIC_BUILTINS was defined it did these two lines also when CPU != powerpc now not sure what happens.